active-measures-module/src/middleware/auth.ts

```
@@ -1 +1,282 @@
+    .catch((error) => {
+      logger.error('Authentication middleware error', { error: error.message });
+      res.status(500).json({ error: 'Authentication error' });
+    });
+}
+
+// Express middleware for authorization
+export function requirePermission(operation: string, classification?: string) {
+  return (req: any, res: any, next: any): void => {
+    const user = req.user;
+    if (!user) {
+      return res.status(401).json({ error: 'Authentication required' });
```

Biome (2.4.6)

[error] The function should not return a value because its return type is void.

(lint/correctness/noVoidTypeReturn)